vcpkg.json

```diff
@@ -40,14 +40,16 @@
       "name": "opencv4",
       "default-features": false,
       "features": [
-        "ffmpeg",
```

Comment:
Then wrong version off ffmpeg was getting installed which cause rocAL build to fail.

Reply:
Okay. Could you please try one other thing? The ffmpeg override for 4.4 really belongs in rocal's vcpkg.json since that's your dependency. Could you try moving the override there and just add ffmpeg as a regular dependency to the server? Ideally, when you include rocal as a feature, it should follow your override for ffmpeg and install whatever when rocal is not present

Reply:
Tried a few things including your suggestion but they didn't work out. 
The ffmpeg should be overridden to 4.4.
In order to avoid this, we need two different vcpkg.json that can be chosen based on the user input.